metplus/util/diff_util.py

```diff
@@ -675,14 +701,15 @@ def _diff_stat_line(compare_a, compare_b, header, print_error=False):
 
     all_good = True
     for index, (col_a, col_b) in enumerate(zip(cols_a, cols_b)):
+        if _is_version_string(col_a) and _is_version_string(col_b):
+            continue
         if _is_equal_rounded(col_a, col_b):
             continue
         all_good = False
         if not print_error:
             continue
 
-        label = f'column {index+2}' if index >= len(header) else header[index]
-        message += f"  Diff in {label}:\n    A: {col_a}\n    B: {col_b}\n"
+        message += f"  Diff in column {index+1}:\n    A: {col_a}\n    B: {col_b}\n"
```

Comment:
Why did this get changed? Previously, it would list the header value instead of just the index. This was very useful in determining which value has changed.